.scripts/package.cmd

```diff
@@ -9,9 +9,14 @@ CALL %~dp0init.cmd
 
 PUSHD %~dp0..
 
-REM TODO: these should be read from the version props file, or similar
-SET Version=8.6.1
-SET Tag=-INTERNALONLY
+REM TODO: these should be read from the version props file, or similar, with the ability to overload via env
+IF NOT DEFINED Version (
+    SET Version=8.6.1
+)
+
+IF NOT DEFINED Tag (
+    SET Tag=-INTERNALONLY
```

Comment:
Is tag an env var that nuget reads when packaging?

Reply:
Yes, but only because we explicitly set it in the nuspec file: https://github.com/VowpalWabbit/vowpal_wabbit/blob/master/cs/cs/cs.nuspec

`    <version>$version$$Tag$</version>`

Reply:
Ah got it, thanks